src/cheri/csv/CHERI_ISA.csv

```diff
@@ -41,12 +41,12 @@
 "C_ADDI16SP_CHERI","✔","✔","","","","✔","✔","Both","","","","✔","","","","","","","","","","","","","C0","","","","ADD immediate to stack pointer in {cheri_int_mode_name}, CADD ","","","","","","","",""
 "C_ADDI4SPN_CHERI","✔","✔","","","","✔","✔","Both","","","","✔","","","","","","","","","","","","","C0","","","","ADD immediate to stack pointer in {cheri_int_mode_name}, CADD ","","","","","","","",""
 "C_MV_CHERI","✔","✔","","","","✔","✔","Both","","","","✔","","","","","","","","","","","","","C2","","","","Integer register move in {cheri_int_mode_name}, capability register move ","","","","","","","",""
-"C.J","✔","✔","","","","✔","✔","Both","","","","✔","","","","","","","","","","","","","C2","","","","Jump to PC+offset, bounds check minimum size target instruction","MODE==D (optional)","","","","","","",""
-"C.JAL","✔","","","","","✔","✔","Both","","","","✔","","","","","","","","","","","","","C2","","","","Jump to PC+offset, bounds check minimum size target instruction, link to cd","MODE==D (optional)","","","","","","",""
-"JAL_CHERI","✔","✔","","","","✔","✔","Both","","","","✔","","","","","","","","","","","","","JAL","","","","Jump to PC+offset, bounds check minimum size target instruction, link to cd","MODE==D (optional)","","","","","","",""
-"JALR_CHERI","✔","✔","","","","✔","✔","Both","","","","","","","","","","","","","","","","","JALR","","","","Indirect jump and link, bounds check minimum size target instruction.  set PCC=unseal(cs1),cd=seal(nextPCC)","MODE==D (optional)","","","","","","",""
-"C.JALR","✔","✔","","","","✔","✔","Both","","","","✔","","","","","","","","","","","","","C2","","","","Indirect jump and link, bounds check minimum size target instruction.  set PCC=unseal(cs1),cd=seal(nextPCC)","MODE==D (optional)","","","","","","",""
-"C.JR","✔","✔","","","","✔","✔","Both","","","","✔","","","","","","","","","","","","","C2","","","","Indirect jump, bounds check minimum size target instruction.  set PCC=unseal(cs1)","MODE==D (optional)","","","","","","",""
+"C.J","✔","✔","","","","✔","✔","Both","","","","✔","","","","","","","","","","","","","C2","","","","Jump to PC+offset","MODE==D (optional)","","","","","","",""
+"C.JAL","✔","","","","","✔","✔","Both","","","","✔","","","","","","","","","","","","","C2","","","","Jump to PC+offset, link to cd","MODE==D (optional)","","","","","","",""
+"JAL_CHERI","✔","✔","","","","✔","✔","Both","","","","✔","","","","","","","","","","","","","JAL","","","","Jump to PC+offset, link to cd","MODE==D (optional)","","","","","","",""
+"JALR_CHERI","✔","✔","","","","✔","✔","Both","","","","","","","","","","","","","","","","","JALR","","","","Indirect jump and link.  set PCC=unseal(cs1),cd=seal(nextPCC)","MODE==D (optional)","","","","","","",""
```

Comment:
Does `JAL_CHERI` not seal the link capability? Why not write it the same way as `JALR_CHERI` i.e. `cd=seal(nextPCC)`?

Reply:
ahhhh you're right - will fix